bib/cmd/bootc-image-builder/main.go

```diff
@@ -30,12 +30,27 @@ import (
 	"github.com/osbuild/bootc-image-builder/bib/internal/util"
 )
 
+type pullPolicy string
+
+func newPullPolicy(s string) (pullPolicy, error) {
+	switch s {
+	case "always", "never":
```

Comment:
What about missing and newer?

Reply:
Thanks good catch re missing. We still need to add more tests and figuring out how to approach newer